python/lsst/ts/audiotrigger/read_serial_temp_scanner.py

```diff
@@ -130,19 +130,20 @@ def __init__(
         self.log = log
         self.simulation_mode = bool(simulation_mode)
         self.sensor_dict = {}
-        self.sample_wait_time = 5
+        self.sample_wait_time = 1
 
         # Fan sensor
         self.fan_sensor = ""
         self.fan_gpio = None
         self.fan_turn_on_temp = 0
         self.fan_turn_off_temp = 0
-        device_id = "/dev/ttyUSB0"
+        device_id = "/dev/ttyUSB_lower_right"
         sensor = common.sensor.TemperatureSensor(log=self.log, num_channels=8)
         callback_func = callback
         log = self.log
         baud_rate = 19200
         kwargs = {
+            "name": "Fan Control Sensors",
```

Comment:
The way `self.serial` is constructed from lines 144 to 162 may lead to errors when constructing the `SerialTemperatureScanner` class. Please extract this to a dedicated method and call that method from the start method. This means that `self.serial` needs to be defined as `self.serial: MockPio | pigpio.pi | None = None` and that you need to add checks to ensure that `self.serial` is not None wherever it is used in the code.

It also means that you need to add _**proper**_ error handling in case constructing the serial fails. Not sure how to signal that it has failed when it does. We will need to talk about that.